### DISCUSSION TITLE
Utilization and padding

### DISCUSSION THREAD

**Discussion by olofk** [The-OpenROAD-Project/OpenROAD#3408] [Q&A] [answered]
I'm doing some quick tests using the scripts in OpenROAD/test to build a design. Using those, I get a surprisingly low utilization, around 50%. Playing around a bit, I realized that decreasing global_place_pad gave me a lot more reasonable results. My question now is if I'm allowed to change that, or if that is a property defined by the PDK and should be left as is. And am I supposed to use the flow script in OpenROAD/test at all, or is that for a particular example? I found that script to be the most self-contained and easiest to extend compared to e.g. ORFS or OpenLANE that relies on a lot of environment variables and custom Python scripts.

**Comment by maliberty** [comment, marked as answer]
You can change it but you might want to start with global_place_density which is the target density.

Note that flow.tcl is for testing and a more complete flow is in ORFS.  If it meets your needs that's ok but it isn't our recommended production flow.

**Reply by olofk** [reply]
Thanks @maliberty . Good to know that flow.tcl at least isn't completely off, so that I can continue using it for quick tests while learning the tools a bit more. Still having way too much fun just clicking around in the GUI :)

And good to know it's allow to change the padding value. global_place_density was already up to 0.99 so I wanted an extra knob to twiddle. Getting routing congestions now instead, which is a lot more reasonable failure mode with this design.
